.goreleaser.yaml

```diff
@@ -0,0 +1,77 @@
+# .goreleaser.yaml
+release:
+  # Repo in which the release will be created.
+  # Default is extracted from the origin remote URL or empty if its private hosted.
+  github:
+    owner: ElliotMJackson
+    name: protoc-gen-validate
+
+  # If set to true, will not auto-publish the release.
+  # Available only for GitHub and Gitea.
+  # Default is false.
+  draft: true
+
+  # Whether to remove existing draft releases with the same name before creating a new one.
+  # Only effective if `draft` is set to true.
+  # Available only for GitHub.
+  #
+  # Default: false.
+  # Since: v1.11.
+  replace_existing_draft: true
+
+  # Useful if you want to delay the creation of the tag in the remote.
+  # You can create the tag locally, but not push it, and run GoReleaser.
+  # It'll then set the `target_commitish` portion of the GitHub release to the value of this field.
+  # Only works on GitHub.
+  #
+  # Default: empty.
+  # Since: v1.11.
+  target_commitish: "{{ .Commit }}"
+
+  # If set to auto, will mark the release as not ready for production
+  # in case there is an indicator for this in the tag e.g. v1.0.0-rc1
+  # If set to true, will mark the release as not ready for production.
+  # Default is false.
+  prerelease: auto
+
+  # What to do with the release notes in case there the release already exists.
+  #
+  # Valid options are:
+  # - `keep-existing`: keep the existing notes
+  # - `append`: append the current release notes to the existing notes
+  # - `prepend`: prepend the current release notes to the existing notes
+  # - `replace`: replace existing notes
+  #
+  # Default is `keep-existing`.
+  mode: keep-existing
+
+  # Header template for the release body.
+  # Defaults to empty.
+  header: |
+    ## Some title ({{ .Date }})
+
+    Welcome to this new release!
+
+  # Footer template for the release body.
+  # Defaults to empty.
+  footer: |
+    ## Thanks!
+
+    Those were the changes on {{ .Tag }}!
```

Comment:
Can we use the release notes template from connect-go? Github supports generating release notes automatically now, and they do a nice job highlighting new contributors.

(Fine to leave this config, or replace it with a message that these notes are temporary and should be replaced with GH's autogen.)